doc/glossary/index.rst

```diff
@@ -31,3 +31,374 @@ Glossary
    Counter
         The token property ``count`` in privacyIDEA is used to calculate the :term:`OTP Value`
         using the HMAC-type algorithms HOTP or TOTP.
+
+   WebUI
+        privacyIDEA comes with a web-based user interface which is used to manage and
+        configure the privacyIDEA server.
+
+        It is also used a self-service portal for the average user, who manages his own tokens.
+        This section gives an overview on the interface and links the respective sections in the documentation.
+
+        * :ref:`dashboard`
+        * :ref:`tokensview`
+        * :ref:`usersview`
+        * :ref:`machines`
+        * :ref:`config`
+        * :ref:`audit`
+        * :ref:`components`
+
+   Time Step
+        A TOTP token can have a timestep of 30/60 seconds. It can still be used, if this 30/60 seconds are over.
+        Technically the timestep is the divider by which the seconds since 1.1.1970 (unix system time)
+        are divided to calculate the OTP value.
+
+        The ``timestep`` is different to the :ref:`Time Window`.
+
+   Time Window
+        Timewindow in which the given OTP value is valid for authentication.
+        ``Timestep and timewindow`` are completely similar to the ``counter and countwindow`` (:ref:`tokeninfo`) of HOTP tokens.
+
+   Revoked Token
+        Tokens can be ``revoked``. Usually this means the token is ``disabled and locked``. A locked token can
+        not be modified anymore. It can only be deleted. Certain token types like certificate may
+        define special actions when revoking a token.
+
+   Orphaned Token
+        An orphaned token means, that it has a user assigned, but the user does not exist in the
+        user store (anymore).
+
+   Disabled Token
+        Tokens can be ``disabled``. Disabled tokens still belong to the assigned user but those tokens
+        can not be used to authenticate. ``Disabled tokens can be enabled again``.
```

Comment:
Just technically: The formatting makes this text a "command" format. You prabably want to use:

~~~~
*Disabled tokens can be enabled again*.
~~~~

Reply:
done in 0dd7436e2018a6334c568484855c62c2f2a2e0da.